src/PlutusCore/Assembler/EntryPoint.hs

```diff
@@ -40,6 +40,7 @@ newtype Verbose = Verbose Bool
 
 data Command
   = CommandAssemble
+    Bool -- rather or not to use shrinker
```

Comment:
I think it would be useful to respect `--shrinking` on `run` and `eval` commands too.